doc/datavolumes.md

```diff
@@ -0,0 +1,136 @@
+# Data Volumes
+
+## Introduction
+Data Volumes(DV) are an abstraction on top of Persistent Volume Claims(PVC) and the Containerized Data Importer(CDI). The DV will monitor and orchestrate the upload/import of the data into the PVC. Once the process is completed, the DV will be in a consistent state that allow consumers to make certain assumptions about the DV in order to progress their own orchestration.
+
+Why is this an improvement over simply looking at the state annotation created and managed by CDI? For one, CDI has different state annotations for different operations (import vs cloning vs upload), where DVs have a single state field. This makes it a lot easier to check that one field vs trying to map all the different annotations in your process. Also DVs will always have a status field, where PVCs may or may not have a status annotation, so you will have to account for that as well.
+
+### Status phases
+The following statuses are possible.
+* 'Blank': No status available.
+* Pending: The operation is pending, but has not been scheduled yet.
+* PVCBound: The PVC associated with the operation has been bound.
+* Import/Clone/UploadScheduled: The operation (import/clone/upload) has been scheduled.
+* Import/Clone/UploadInProgress: The operation (import/clone/upload) is in progress.
+* Succeeded: The operation has succeeded.
+* Failed: The operation has failed.
+* Unknown: Unknown status.
+
+## Import Data Volumes
```

Comment:
Maybe Title the "HTTP Source" and the clone one as "PVC Source"

Behind the scenes we know there are import and clone controllers involved, but from a DataVolume API perspective there are only sources.

Reply:
Yeah sounds good, will change.